### DISCUSSION TITLE
Save in home

### DISCUSSION THREAD

**Discussion by MTollurido** [ltsp/ltsp#930] [Q&A]
Good evening,  I'm using ltsp without chroot in an ubuntu environment. I noticed that sometimes it is not possible to save files in the home dir of the (fat) client. It looks like loosing the access to the server. Also the home dir gets empty until reboot. Any ideas? Thanks a lot. Mauro

**Comment by stevehol** [comment]
I also have this issue, but didn't notice your post before writing my own: https://github.com/ltsp/ltsp/discussions/942 
Did you find a workaround?
